openmp/libomptarget/plugins-nextgen/common/include/PluginInterface.h

```diff
@@ -904,6 +910,9 @@ struct GenericDeviceTy : public DeviceAllocatorTy {
   UInt32Envar OMPX_MinThreadsForLowTripCount =
       UInt32Envar("LIBOMPTARGET_MIN_THREADS_FOR_LOW_TRIP_COUNT", 32);
 
+  BoolEnvar OMPX__ReuseBlocksForHighTripCount =
```

Comment:
One `_` is enough. We need a comment here explaining the meaning. Also need to add it to the docs.
Finally, can you add this into the test that checks for the trip count adjustment as a new run line with the variable set?

Reply:
fixed